translator/translate/logs/logs_collected/files/collect_list/collect_list_test.go

```diff
@@ -310,6 +310,22 @@ func TestTimestampFormat_Template(t *testing.T) {
 	assert.Equal(t, time.Date(0, 8, 9, 20, 45, 51, 0, time.Local), parsedTime)
 }
 
+func TestTimestampFormat_InvalidRegex(t *testing.T) {
+	translator.ResetMessages()
+	r := new(TimestampRegax)
```

Comment:
I didn't notice that the name of this is misspelled. Looks like it was always like this. Not important to fix in this PR but a typo we should fix at some point